freud/data.py

```diff
@@ -65,6 +65,12 @@ def generate_system(self, num_replicas=1, scale=1, sigma_noise=0, seed=None):
             tuple (:class:`freud.box.Box`, :class:`np.ndarray`):
                 A system-like object (see
                 :class:`~freud.locality.NeighborQuery.from_system`).
+
+        Note:
+            Positions are generated in the order of the instance's
+            ``basis_positions``, contiguously. The first :math:`N_{replica}`
```

Comment:
```suggestion
            ``basis_positions``. The first :math:`N_{replica}`
```
I don't think using the word contiguously here is really necessary. If anything, I think it makes it harder to understand what this note is saying.